WrocRide/Controllers/UserController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using WrocRide.Entities;
+using WrocRide.Models;
+using WrocRide.Services;
+
+namespace WrocRide.Controllers
+{
+    [ApiController]
+    [Route("api/me")]
+    public class UserController : ControllerBase
+    {
+        private readonly IUserService _userService;
+
+        public UserController(IUserService userService)
+        {
+            _userService = userService;
+        }
+
+        [HttpGet]
+        public ActionResult<UserDto> Get()
+        {
+            var result = _userService.GetUser();
+
+            if (result == null) { return NotFound(); }
```

Comment:
Tutaj nie musisz tego sprawdzać bo to samo robisz w serwisie, wiec ta linia do usunięcia.